cannot find module 'Item.json'

There is no item.json file in the folder. How to create?


`> node-dbc-read@1.0.0 start /home/ubuntu/tools/node-dbc-reader
> node src/index.js "--search=[44641].includes({*})" "--columns=ID" "Item"

Reading Item.dbc file
(node:185349) UnhandledPromiseRejectionWarning: Error: Cannot find module '/home/ubuntu/tools/node-dbc-reader/src/schemas/azerothcore/item.json'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at DBC.getSchema (/home/ubuntu/tools/node-dbc-reader/src/dbc.js:41:17)
    at DBC.read (/home/ubuntu/tools/node-dbc-reader/src/dbc.js:78:22)
    at DBC.toJSON (/home/ubuntu/tools/node-dbc-reader/src/dbc.js:47:14)
    at extractDBC (/home/ubuntu/tools/node-dbc-reader/src/index.js:101:14)
    at Command.actionCommand (/home/ubuntu/tools/node-dbc-reader/src/index.js:146:11)
    at Command.listener [as _actionHandler] (/home/ubuntu/tools/node-dbc-reader/node_modules/commander/index.js:922:31)
(node:185349) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:185349) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
`

--- a/src/schemas/azerothcore/item.json
+++ b/src/schemas/azerothcore/item.json
@@ -1,0 +1,34 @@
+[
+  {
+    "name": "ID",
+    "type": "int"
+  },
+  {
+    "name": "ClassID",
+    "type": "int"
+  },
+  {
+    "name": "SubclassID",
+    "type": "int"
+  },
+  {
+    "name": "Sound_Override_Subclassid",
+    "type": "int"
+  },
+  {
+    "name": "Material",
+    "type": "int"
+  },
+  {
+    "name": "DisplayInfoID",
+    "type": "int"
+  },
+  {
+    "name": "InventoryType",
+    "type": "int"
+  },
+  {
+    "name": "SheatheType",
+    "type": "int"
+  }
+]

